zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -67,6 +68,10 @@ class EcommerceSpiderParams(BaseSpiderParams):
     )
 
 
+class EcommerceSpiderParams(EcommerceCrawlStrategyParam, BaseSpiderParams):
+    pass
```

Comment:
Is the order of params the same as before, i.e. if you add a mixin in front, the param is added to the end?

Reply:
In a previous example the order was the opposite - URL field first, etc. How does it work?

Reply:
https://github.com/zytedata/zyte-spider-templates/pull/46/commits/af9fe4d71bbf39c0e15f5d9001f82e10431c57f2